lib/std/crypto/25519/ed25519.zig

```diff
@@ -503,10 +504,17 @@ test "signature" {
     try std.testing.expectError(error.SignatureVerificationFailed, sig.verify("TEST", key_pair.public_key));
 }
 
+fn initTestKeypair(comptime Scheme: type, rng: *std.Random.DefaultPrng) !Scheme.KeyPair {
+    var seed: [Scheme.KeyPair.seed_length]u8 = undefined;
+    rng.fill(&seed);
+    return try Scheme.KeyPair.createDeterministic(seed);
+}
+
 test "batch verification" {
+    var prng = std.Random.DefaultPrng.init(std.testing.random_seed);
```

Comment:
People frequently look at tests to understand how to use things. So maybe use `DefaultCsprng` instead, and rename `prng` to `test_prng` to make it clear that this code is only for testing?